modules/cnf-du-annotating-workloads.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-annotating-workloads_{context}"]
+
+= Annotating workloads
+
+Two annotations must be available to isolate a workload into partitioned resources. Annotate the containing namespace with `workload.openshift.io/allowed` and the pod
+with `target.workload.openshift.io`.
+
+Required OCP platform pods and namespaces are already annotated. To add extra pods to the partition, such as supported day 2 operators or any user-added catalog sources, annotate both the namespace and pod on creation.
+
+The namespace annotation allows cluster administrators to control which workloads have access to the partitioned resources. Only pods in namespaces containing the annotation `workload.openshift.io/allowed: {<workload-type> [, <workload-type]}` will be considered. Annotated pods that are contained in an unannotated Namespace will fail to admit to the cluster. In this case, an error is raised indicating the pod is forbidden because the namespace does not allow the `workload type <workload-type>`.
+
+Pods that should be isolated on the partitioned resources must be annotated with `target.workload.openshift.io/<workload-type>`. In the current release the only supported value for <workload-type> is `management`. Future releases may support multiple values for <workload-type>. The value of the annotation is a JSON struct as shown below.
```

Comment:
The word "isolated" here (and in the preceding para too) is confusing *me*, and maybe just me.  Isolated from what?  My suspicion is that you can omit "isolated" as a term and limit the terminology to "workload partitioning".

Can you revise the first sentence to use active voice?  Maybe if it started with "To run pods on the reserved CPUs..." or "To run pods on a workload partition..."

nit: Change "may" to "can" or "might" according to ISG around p362.

nit: "below" has similar ISG guidance as "above". I suggest "...as shown in the following code block."  See ISG around p279 for more info.